tools/muscle/patches/5.1.0.path

```diff
@@ -0,0 +1,106 @@
+diff --git a/src/Makefile b/src/Makefile
```

Comment:
Please rename this file `5.1.0.patch` (missing `c` in file extension`.
Otherwise biowasm doesn't use this file as a  patch

Reply:
sorry, I missed the suffix~